components/AppCard.tsx

```diff
@@ -15,7 +15,7 @@ export type AppCardProps = {
 export const AppCard = ({ name, icon, url, paid }) => {
   return (
     <a
-      key={url}
+      key={url + name}
```

Comment:
I think the following should be more in line with the rest of the codebase regarding string interpolation, also makes it less likely for the start of the name to mess with the end of the URL:

```suggestion
      key={`${url} ${name}`}
```